man/udevadm.xml

```diff
@@ -961,6 +961,16 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><option>--changed</option></term>
+          <listitem>
+            <para>When specified, the command wait for devices being changed instead of created or
```

Comment:
"changed" here is ambiguous. And, from the code, this implies "removed", even if the device does not exist when the command is invoked.

What is the use case here??

Reply:
Thanks @yuwata for the review. I've pushed a new rev with suggested changes and more justification in the commit message.

Reply:
I just realized I missed changes to TEST-17-UDEV. Please wait to review again until I push another rev.